dlt/common/storages/fsspec_filesystem.py

```diff
@@ -276,6 +281,7 @@ def glob_files(
 
     bucket_path = bucket_url_parsed._replace(scheme="").geturl()
     bucket_path = bucket_path[2:] if bucket_path.startswith("//") else bucket_path
+    bucket_path = bucket_path.split("?")[0]
```

Comment:
Query params (`RootId=xxx`) should be dropped before using the `bucket_path` further